src/content/docs/dropins/cart/cart-functions.mdx

```diff
@@ -298,118 +394,147 @@ type StoreConfigModel = {
 };
 ```
 
+### Usage
+
+To get information about a store's configuration:
+
+```js
+import { getStoreConfig } from '@/cart/api/getStoreConfig';
+
+getStoreConfig();
+```
+
 ## initializeCart
 
-A function that initializes either a guest or customer Cart.
+The `initializeCart` function initializes a guest or customer cart.
```

Comment:
**Suggestion:** Might be relevant to mention that this function is automatically called during the initialize phase of a dropin's lifecycle, and there would be no need to call this manually. It also emits the `cart/initialized` and `cart/data` events.